PSSwagger/SwaggerUtils.psm1

```diff
@@ -987,6 +987,29 @@ function Get-ParamType
                     Write-Warning -Message $Message
                 }
             }
+            elseif($ParameterJsonObject.Type -eq 'string') {
+                if((Get-Member -InputObject $ParameterJsonObject.AdditionalProperties -Name 'Type') -and
+                   ($ParameterJsonObject.AdditionalProperties.Type -eq 'array'))
+                {
+                    if((Get-Member -InputObject $ParameterJsonObject.AdditionalProperties -Name 'Items') -and
+                       (Get-Member -InputObject $ParameterJsonObject.AdditionalProperties.Items -Name 'Type') -and
+                       $ParameterJsonObject.AdditionalProperties.Items.Type)
+                    { 
+                        $ItemsType = Get-PSTypeFromSwaggerObject -JsonObject $ParameterJsonObject.AdditionalProperties.Items
```

Comment:
Does this handle $ref?

Reply:
UnsupportedSwaggerProperties warning message should be thrown for AdditionalProperties.Items.$ref.

Reply:
Added support for AdditionalProperties.Items.$ref .